src/main/java/com/hubspot/jinjava/lib/filter/RenderFilter.java

```diff
@@ -38,4 +46,20 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
     }
     return interpreter.render(Objects.toString(var));
   }
+
+  private void removeModuleErrors(JinjavaInterpreter jinjavaInterpreter) {
+    Stack<TemplateError> validErrors = new Stack<>();
+    while (jinjavaInterpreter.getLastError().isPresent()) {
+      TemplateError te = jinjavaInterpreter.getLastError().get();
+      jinjavaInterpreter.removeLastError();
+      if (te.getReason() == ErrorReason.DISABLED && te.getMessage().contains("module")) {
```

Comment:
Let's get rid of the `module` terminology since that's just for HubL. I think it would be fine to remove the module terminology.